pydocumentdb/execution_context/execution_dispatcher.py

```diff
@@ -90,7 +91,7 @@ def fetch_next_block(self):
         return self._execution_context.fetch_next_block()        
         
     def _is_partitioned_execution_info(self, e):    
-        return e.status_code == 400 and e.sub_status == 1004
```

Comment:
Can you add SubStatusCodes as well, in the constants file? And change references to 1004 and other sub status codes to constants.